### ISSUE TITLE
Certain sound effects aren't working

### ISSUE THREAD

**Issue by MicroVirus** [the3dfxdude/7kaa#93]
I noticed in the current developer version that certain sound effects, like the catapult and unicorn projectiles, aren't working anymore.
My guess is it has to do with the resource files somehow.

Update: [forum report](https://www.7kfans.com/forums/viewtopic.php?p=7068#p7068) on Norman sword sound also not working for 2.14.7.

**Comment by the3dfxdude** [comment]
This is also working here. But I don't exactly discount this problem though. While not much editing of the resource files have occurred, one thing I recently noticed is how the basic race identifiers are used in many places of the code. With the introduction of gettext support, the initial implementation had read in the resource files, then translated some of the fields, and stored it back into the structure. This is a potential huge issue, since the code does do direct string compares to line up resources for nationality specific resources. I wouldn't doubt this is how the sounds for a unit are found, but I haven't checked this.

Anyway, the potential problem with overwriting the data, and the strategy behind gettext (suggests full sentence translations, not piece by piece) does not lend to this having been the best choice. I have been working about backing out the changes and restore the original code, with new code to extract the resource data for translation, but I got delayed due to lack of time. It's almost done, except the canned gettext Makefile got me sidetracked.
